omero/sysadmins/unix/server-centos7-ice36.txt

```diff
@@ -87,12 +87,14 @@ Configure:
     :start-after: #end-release-ice36
     :end-before: #end-step04
 
-Installing OMERO.web
----------------------
+Installing and Running OMERO.web
+--------------------------------
 
-OMERO.web is deployed using Nginx, for more details
+OMERO.web is deployed using Nginx, for more details on how to install
+and run the OMERO.web client
 see :doc:`install-web/walkthrough/omeroweb-install-centos7-ice3.6`.
 
+
 Running OMERO
```

Comment:
Again, maybe 'Running OMERO.server' to be less confusing with '..Running OMERO.web' above